.github/workflows/strr-application-auto-approval-job-cd.yaml

```diff
@@ -0,0 +1,30 @@
+name: STRR APPLICATION AUTO APPROVAL JOB CD
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "jobs/auto-approval/**"
+  workflow_dispatch:
+    inputs:
+      target:
+        description: "Deploy To"
+        required: true
+        type: choice
+        options:
+          - dev
+          - test
+          - sandbox
+          - prod
+
+jobs:
+  strr-pay-cd:
+    uses: bcgov/bcregistry-sre/.github/workflows/backend-cd.yaml@main
```

Comment:
if its a job, needs to use  bcgov/bcregistry-sre/.github/workflows/backend-job-cd.yaml@main